src/main/java/com/pet/bankservice/entity/Transaction.java

```diff
@@ -30,11 +33,28 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @ManyToOne
+    @JoinColumn(name = "from_account_id")
     private Account fromAccount;
     @ManyToOne
+    @JoinColumn(name = "to_account_id")
     private Account toAccount;
+    @Column(name = "date_time")
     private LocalDateTime dateTime;
-    private Double amount;
+    private BigDecimal amount;
     @Enumerated(EnumType.STRING)
     private TransactionType type;
+    private String status;
```

Comment:
let's remove this field or make it en Enum

Reply:
done